src/main.c

```diff
@@ -133,10 +133,36 @@ Please check your configuration file and permissions\
   }
 
   const char *mrl_file = get_most_recent_list_filename (data_dir);
+  if (verbose)
+    printf ("most recent list (mrl file): %s\n", mrl_file);
 
-  if (cli_user_options.want_undo)
+  if (cli_user_options.list_most_recent || cli_user_options.want_undo)
   {
-    undo_last_rmw (st_config_data.st_waste_folder_props_head, &st_time_var, mrl_file, &cli_user_options);
+    char *mrl_contents = get_mrl_contents (mrl_file);
+
+    if (mrl_contents != NULL)
+    {
+      if (cli_user_options.list_most_recent)
```

Comment:
The logic here implies that if you try e.g. `rmw -um` , the `u` option will be silently ignored. Is this the intended behaviour?

Reply:
Now that you mention it, `silently` ignoring probably isn't too user-friendly, is it? I'll make a change.

Reply:
Resolved with https://github.com/theimpossibleastronaut/rmw/pull/285/commits/45a08384ddb622bce3e2d52a5f736df25a6f7c2d